scripts/build.sh

```diff
@@ -0,0 +1,13 @@
+tsc -p tsconfig.json && tsc -p tsconfig.cjs.json
+
+cat >./dist/package.json <<!EOF
+{
+    "type": "module"
+}
+!EOF
+
+cat >./dist/cjs/package.json <<!EOF
+{
+    "type": "commonjs"
+}
```

Comment:
Another option would be to make the project an ESM project and have a `/src/cjs` folder with a `package.json` that defines the `cjs` folder as CJS project like we have in the [chromedriver service](https://github.com/webdriverio-community/wdio-chromedriver-service/tree/main/src/cjs).

Reply:
I tried this approach first and couldn't get it to work with the example app approach I am using.  Given I have three package jsons in the repo now I'm still considering a monorepo-type approach using pnpm workspaces might be better if only to streamline deps & scripts.  The current package and ESM approach works for now so I am inclined to reduce scope and revisit later.

Reply:
I found a way round the issues I was experiencing, the ESM approach is now close to that of the Chromedriver service.